bokeh/models/graphs.py

```diff
@@ -33,22 +31,9 @@ class StaticLayoutProvider(LayoutProvider):
         }
     """)
 
-
-_DEFAULT_NODE_SOURCE = lambda: ColumnDataSource(data=dict(index=[]))
-
-
-_DEFAULT_EDGE_SOURCE = lambda: ColumnDataSource(data=dict(start=[], end=[]))
-
-
-class GraphSource(Model):
-    '''
-
-    '''
-
-    @classmethod
-    def from_networkx(cls, graph, layout_function, **kwargs):
+def from_networkx(graph, layout_function, **kwargs):
```

Comment:
debating whether this should be `from_networkx` or just `networkx` We can decide before the release tho